docker-compose.yml

```diff
@@ -1,21 +1,124 @@
 version: '2'
 services:
- db:
+  db:
     image: mysql
     environment:
       - MYSQL_ROOT_PASSWORD=root
     volumes:
       - ./database_setup.sql:/docker-entrypoint-initdb.d/schema.sql:ro
+      - db_data:/var/lib/mysql
     ports:
-      - 3306
- groot:
-   build: .
-   command: /bin/bash -c "./docker_go_symlink.sh && ./dev_spinup.sh"
-   working_dir: /groot-deploy
-   volumes:
-     - .:/groot-deploy
-   ports:
-     - "8000:8000"
-     - "5000:5000"
-   depends_on:
-     - db
+      - "3306:3306"
+    restart: always
+
+  groot:
+    build: groot
+    ports:
+      - "8000:8000"
+    volumes: 
+      - clients_db:/var/groot/
+    restart: always
+
+  groot-credits-service:
+    build: groot-credits-service
+    ports:
+      - "8765:8765"
+    depends_on:
+      - db
+      - groot
+    restart: always
+
+  groot-desktop-frontend:
+    build: groot-desktop-frontend
+    depends_on:
+      - groot
+      - groot-events-service
+      - groot-users-service
+      - groot-groups-service
+      - groot-quotes-service
```

Comment:
not entirely sure what depends_on does but are these the correct service dependencies?